packages/sanity/src/core/releases/tool/detail/ReleaseSummary.tsx

```diff
@@ -74,9 +74,9 @@ export function ReleaseSummary(props: ReleaseSummaryProps) {
       if (!isBundleDocumentRow(rowProps.datum)) return null
       if (rowProps.datum.isPending) return null
 
-      return <DocumentActions document={rowProps.datum} />
+      return <DocumentActions document={rowProps.datum} releaseName={release.metadata.title} />
```

Comment:
I still think it's a bit confusing that we're mixing _name_ and _title_. We do have a concept of release name, which  is different from the title we display in various places. If this is intentional, could we have a comment explaining the rationale?